### ISSUE TITLE
Good program!

### ISSUE THREAD

**Issue by mrkaban** [DevComPack/setupmaker#6]
Hello, I wanted to thank you for the good program! it really is quality! I liked it and I added it to my site about free software, here is the [link](https://xn--90abhbolvbbfgb9aje4m.xn--p1ai/%D1%83%D1%82%D0%B8%D0%BB%D0%B8%D1%82%D1%8B/%D0%BF%D1%80%D0%BE%D1%87%D0%B5%D0%B5/dcpsetupmaker-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-%D1%83%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D1%89%D0%B8%D0%BA%D0%BE%D0%B2-%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC.html).

The only thing I did not understand was how to make the shortcut to the executable file on the desktop? is it necessary to do something with the groups?

Need help translating into Russian? I'm not a professional, but I'm ready to help.

**Comment by saidelimam** [comment]
Hello, I'm glad you found it useful! Thank you for your feedback :)

For the shortcut you only need to select the executable pack and then enable the shortcut option in the properties below. No groups necessary.

Yes that would be awesome if you could help with the Russian! You just need to duplicate a new file in this directory https://github.com/DevComPack/setupmaker/tree/master/res/langpacks and make a PR for it. I will merge it ASAP.
The file name should be `rus.xml` and contains the same content as other files except for the `txt` attribute which is translated to Russian. Don't hesitate to ask if you need more help.

Have a good day 👍

**Comment by mrkaban** [comment]
I began to translate, one of these days I will finish. Can I check the translation in the program before publication? Maybe I want to correct something in translation.

**Comment by mrkaban** [comment]
I can not upload the file, I do not have rights, so I attach it to the message. I ask to show the preliminary version for checking the translation and the subsequent correction of individual elements.
[rus.zip](https://github.com/DevComPack/setupmaker/files/1767605/rus.zip)

**Comment by saidelimam** [comment]
Yes you should make a Pull Request with the new file, not directly push it.
Ok I'll add it, you can already see the result by adding your
file to the DCP folder/res and then using it from the program in the custom
language pack field.

However this is a translation for the compiled installers. If you meant to
translate DCP itself you'll have to edit in the source code (Bxml files)
and rebuild a release :)

**Comment by mrkaban** [comment]
I did not understand how to check the translation. It did not turn out to switch to Russian. Well, okay. Write please, as you publish the version with the Russian language, I will add a note to the description of the program.
